Pocket/Pocket/Protocols/PocketRequestManagerProtocol.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  PocketRequestManagerProtocol.swift
+//  Pocket
+//
+//  Created by Pabel Nunez Landestoy on 6/7/18.
+//  Copyright © 2018 Pocket Network. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+public typealias SendRequestHandler = (_: JSON, _: Error?) -> Void
+
+public protocol PocketRequestManagerProtocol: AnyObject {
```

Comment:
Does this need to be a protocol?

Reply:
We wanted developers to be able to create their own Request Managers if they wanted to, I know there are extensions, but in case you wanted to create your own class optimized for your use case.